app/models/custom_field/order_statements.rb

```diff
@@ -72,8 +72,15 @@ def select_custom_value_as_string
   end
 
   def select_custom_values_as_group
+    aggr_sql =
+      if OpenProject::Database.mysql?
+        "GROUP_CONCAT(cv_sort.value SEPARATOR '.')"
+      else
+        "string_agg(cv_sort.value, '.')"
+      end
+
     <<-SQL
-      COALESCE((SELECT string_agg(cv_sort.value, '.') FROM #{CustomValue.table_name} cv_sort
```

Comment:
@machisuji Shame! `string_agg` isn't cross compatible but it appears this line was never called :D